components/LandingPage.tsx

```diff
@@ -2,97 +2,216 @@ import * as React from 'react'
+              alt="Curriculum"
+              style={{ height: '170px' }}
+            ></img>
+            <h4 className="mt-3 font-weight-bold">Modern Curriculum</h4>
+            <p className="mt-4 font-weight-light">
+              First the curriculum will help you achieve mastery in JavaScript.
+              Using JavaScript, you will learn about different branches of
+              software engineering: websites, mobile applications, servers, and
+              machine learning.
+            </p>
+          </div>
+          <div className="col-md-4">
+            <img
+              src="choose-02.svg"
+              alt="Support"
+              style={{ height: '170px' }}
+            ></img>
+            <h4 className="mt-3 font-weight-bold">Support</h4>
+            <p className="mt-4 font-weight-light">
+              One of the most important skills in a challenging workplace is
+              knowing how to ask for help. Our curriculum is intentionally
+              difficult and we have dedicated mentors to answer your questions
+              and guide you how to communicate effectively.
+            </p>
+          </div>
+          <div className="col-md-4">
+            <img
+              src="choose-03.svg"
+              alt="Curriculum Image"
+              style={{ height: '170px' }}
+            />
+            <h4 className="mt-3 font-weight-bold">Work Experience</h4>
+            <p className="mt-4 font-weight-light">
+              Once you have completed our curriculum and learned to communicate
+              effectively, you will be invited to contribute to this open-source
+              project to improve the learning experience for future students.
+            </p>
+          </div>
         </div>
       </div>
-      <div className="row mt-5">
-        <div className="col-md-10 offset-md-1">
-          <h4 className="font-weight-light" style={{ lineHeight: 1.5 }}>
+      <div
+        className="container-fluid py-5"
+        style={{ background: 'rgb(84,64,216)', color: 'white' }}
+      >
+        <div className="container mt-5">
+          <h1
+            id="journey"
+            className="mt-5 display-4 font-weight-bold text-center"
+          >
+            Your Learning Journey
+          </h1>
+          <h4
+            className="mt-5 font-weight-light col-md-8 offset-md-2 pb-5 text-center"
+            style={{ lineHeight: 1.5 }}
+          >
             Our learning process is interactive and follows the same practice as
             a well functioning engineering team. From the first line of code you
             write, you will be code reviewed by engineers to ensure you are
-            always following industry best practices.
+            always following industry best practice
           </h4>
+          <div className="row mt-5 pb-5">
+            <div className="col-md-5 offset-md-2 order-md-2 text-center">
+              <img src="journey-01.svg" className="mb-3 img-fluid"></img>
```

Comment:
I think screen readers might read "journey-01.svg". If you want to silence this, you can add an empty alt attribute like this: `alt=""`

Quote: "You should use CSS background images for decorative images, but if you must use HTML, add a blank alt="". If the image isn't part of the content, a screen reader shouldn't waste time reading it."

source: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML

Reply:
I decided to add `alt` as the same as the section header. I can't seem to decide whether it should be `alt=""` or `alt="title of section"`. 

I didn't want to change it use background image since each image has a different height and I would have to specify the height of each div.

Reply:
I'm not 100% sure either, but I think `alt=""` might be good because I think the images seem like redundant content to a screen reader if they're the same as the headings.

I think it's fine to not use a background image and instead use `alt=""`.

Reply:
Thanks. I just changed to `alt=""`